foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_8_0/OrphedomosToFabric8Migration.java

```diff
@@ -253,24 +324,104 @@ private static String getConfigCode(String indent){
                 StringUtils.repeat(indent, 4) + "</executions>\n";
     }
 
-    /**
-     * Function to check whether a given pom file has its packaging type set to Orphedomos
-     */
-    private boolean containsOrphedomosPackaging(File file){
-        boolean shouldExecute = false;
-        try {
-            List<String> orphedomosPackaging = getRegExCaptureGroups(EXTRACT_PACKAGING_REGEX, file);
-            shouldExecute = CollectionUtils.isNotEmpty(orphedomosPackaging);
-        } catch (IOException e){
-            throw new BatonException("Unable to determine if Orphedomos packaging migration must be executed", e);
+    private void updateModWithDeleteOrphedomosConfig(Plugin fabric8Plugin, PomModifications removeOldConfig) {
+        InputLocation startConfigLocation = fabric8Plugin.getLocation(CONFIGURATION + LocationAwareMavenReader.START);
+        InputLocation endConfigLocation = fabric8Plugin.getLocation(CONFIGURATION + LocationAwareMavenReader.END);
+        PomModifications.Deletion pomDeletePreviousConfig = new PomModifications.Deletion(startConfigLocation, endConfigLocation);
+        removeOldConfig.add(pomDeletePreviousConfig);
+    }
+
+    private void updateModWithDeleteSingleLineVersionTag(Plugin fabric8Plugin, PomModifications removeOldConfig) {
+        InputLocation startVersionLocation = fabric8Plugin.getLocation(VERSION + LocationAwareMavenReader.START);
+        InputLocation endVersionLocation = fabric8Plugin.getLocation(VERSION + LocationAwareMavenReader.END);
+
+        PomModifications.Deletion pomDeletePreviousVersion = new PomModifications.Deletion(startVersionLocation, endVersionLocation);
+        removeOldConfig.add(pomDeletePreviousVersion);
+    }
+
+    private boolean executeProfileMigration(File file, Plugin profilePlugin, String profileId) {
```

Comment:
this function is the main addition to the migration, the rest of the changes in this file are largely refactoring to make shared helper functions available between the build config and profile migrations